### ISSUE TITLE
mpr-core depends on SNAPSHOT version of flow-server

### ISSUE THREAD

**Issue by mvysny** [vaadin/multiplatform-runtime#67]
mpr-core 2.0.1 pom.xml depends on a SNAPSHOT version of flow-server. That later wreaks havoc in Gradle since Gradle resolves version conflicts differently than Vaadin and prefers highest version:

```
|    |    |    |    |    +--- com.vaadin:mpr-v8:2.0.1
|    |    |    |    |    |    \--- com.vaadin:mpr-core:2.0.1
|    |    |    |    |    |         +--- com.vaadin:flow-html-components:3.0-SNAPSHOT
|    |    |    |    |    |         |    +--- com.vaadin:flow-server:3.0-SNAPSHOT (*)
|    |    |    |    |    |         |    \--- com.vaadin:flow-data:3.0-SNAPSHOT
|    |    |    |    |    |         |         +--- com.vaadin:flow-server:3.0-SNAPSHOT (*)
|    |    |    |    |    |         |         \--- javax.validation:validation-api:2.0.0.Final -> 2.0.1.Final
|    |    |    |    |    |         +--- com.vaadin:flow-client:3.0-SNAPSHOT
|    |    |    |    |    |         +--- com.vaadin:license-checker:1.0.0
|    |    |    |    |    |         |    \--- com.google.gwt:gwt-elemental:2.8.0 (*)
|    |    |    |    |    |         \--- commons-io:commons-io:2.5 -> 2.6
```

**Comment by mvysny** [comment]
mpr-core 2.0.2 depends on flow-server 2.2-SNAPSHOT.

**Comment by mehdi-vaadin** [comment]
I added the `flow.version` parameter to the MPR release build in TeamCity. It should fix the problem. Should be verified when the next version of MPR is released.

**Comment by mehdi-vaadin** [comment]
It's fixed in the latest release of MPR which is 2.0.3.
